dotCMS/src/main/java/com/dotcms/contenttype/business/sql/RelationshipSQL.java

```diff
@@ -20,7 +20,7 @@ public static RelationshipSQL getInstance() {
 
 	public static final String SELECT_ALL_FIELDS = "select inode, parent_structure_inode, child_structure_inode, "
```

Comment:
I think this query needs to be updated since is selecting from inode

Reply:
It's selecting from relationship table only